### ISSUE TITLE
Could this be updated to work with new patch?

### ISSUE THREAD

**Issue by dkstrong** [icza/s2prot#53]
A new patch for SC2 came out (5.0.12.9115) with balance changes, and new replays won't work with this tool anymore. Is it possible to get this tool updated? I would contribute a PR for the new build but I'm not sure how the code is generated for the new protocols.

**Comment by icza** [comment]
Added the new protocol files, see 2f649b292b4ac2066f9e7bf8e3a5db534ba00dd1

**Comment by icza** [comment]
There you go, released v1.5.1 including support for all replays up to the latest.

https://github.com/icza/s2prot/releases/tag/v1.5.1

**Comment by Izopi4a** [comment]
in case someone like me wonders around, or need it in the future in case there is no release

``` 
git clone... 
s2prot/cmd/s2prot# go run s2prot.go -details _PATH_TO_FILE.SC2Replay
```
